src/python/pants/backend/python/subsystems/pytest_runner_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Replaces the old test in src/python/pants/backend/python/subsystems/pytest_test.py